cve_bin_tool/cve_scanner.py

```diff
@@ -64,13 +64,27 @@ def get_cves(self, product_info: ProductInfo, triage_data: TriageData):
         Example:
             nvd.get_cves('haxx', 'curl', '7.34.0')
         """
+
+        # Prevent any queries resulting in CVEs with UNKNOWN score value
+        # being reported
+        if self.score > 10:
```

Comment:
There's a potential joke here about cve scores going up to 11, Spinal Tap style....
https://en.wikipedia.org/wiki/Up_to_eleven